src/Compilers/CSharp/Test/Semantic/Semantics/RefLocalTests.cs

```diff
@@ -1,1434 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
-using Microsoft.CodeAnalysis.CSharp.Syntax;
-using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
-using Microsoft.CodeAnalysis.Test.Utilities;
-using Roslyn.Test.Utilities;
-using Xunit;
-
-namespace Microsoft.CodeAnalysis.CSharp.UnitTests
-{
-    public class RefLocalTests : CompilingTestBase
```

Comment:
Please delete these files.
I will go through the tests and figure what is salvageable and make a separate PR including those with in files with different names (so that there is no conflict).